Emby.Server.Implementations/Library/Resolvers/Audio/MusicAlbumResolver.cs

```diff
@@ -92,13 +93,22 @@ public bool IsMusicAlbum(string path, IDirectoryService directoryService)
         /// Determine if the supplied resolve args should be considered a music album.
         /// </summary>
         /// <param name="args">The args.</param>
-        /// <returns><c>true</c> if [is music album] [the specified args]; otherwise, <c>false</c>.</returns>
```

Comment:
This should stay the same, consistent with official docs

Reply:
Didn't know the semicolon came from official guidelines. I'll chgange it.